cardano_node_tests/tests/kes.py

```diff
@@ -0,0 +1,39 @@
+"""Functionality for KES key used in multiple tests modules."""
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+
+def check_kes_period_info_result(kes_output, expected_scenario):
+    """Check output from kes-period-info command."""
+    output_scenario = None
+
+    # expected scenario: all_valid
+    if kes_output.get("valid_counters") and kes_output.get("valid_kes_period"):
+        # check metrics, metrics dict are only returned by the command if everything is valid
+        metrics = kes_output.get("metrics")
+        node_state_counter = metrics.get("qKesNodeStateOperationalCertificateNumber")
+        on_disk_counter = metrics.get("qKesOnDiskOperationalCertificateNumber")
+        start_kes_period = metrics.get("qKesStartKesInterval")
+        end_kes_period = metrics.get("qKesEndKesInterval")
+        current_kes_period = metrics.get("qKesCurrentKesPeriod")
+
+        # check if the metrics match with the expected scenario
+        if (
+            node_state_counter <= on_disk_counter
+            and start_kes_period <= current_kes_period <= end_kes_period
+        ):
+            output_scenario = "all_valid"
```

Comment:
Please use constants for the scenarios names both here and in tests. You can create "enum" like
```python
class KesScenarios:
    ALL_VALID = "all_valid"
    ...

# use it like
output_scenario = KesScenarios.ALL_VALID
```